docs/modern/ConversionPlaybook.md

```diff
@@ -17,6 +17,8 @@ Install the latest version of Relay from the [getting started guide](./relay-mod
 
 Start converting your components and mutations to use the Relay Modern APIs from the `'react-relay/compat'` module (`createFragmentContainer`, `createRefetchContainer`, `createPaginationContainer`, `commitMutation`). It will be easier to go from the leaf components up. The [conversion scripts](https://github.com/relayjs/relay-codemod) should make this step less tedious.
 
+Note that if your container requires variables that are determined at runtime, [you must use <QueryRenderer> to set the variables](https://github.com/facebook/relay/issues/1770#issuecomment-302175983).
```

Comment:
I agree that this is worth documenting, though I would make two changes:
- Move this to a separate section about query variables (toward the bottom, after introducing QueryRenderer for example).
- Inline that comment rather than link to it.